src/CommitmentsV2/SFA.DAS.CommitmentsV2/Data/ProviderCommitmentsDbContext.cs

```diff
@@ -32,6 +32,7 @@ public class ProviderCommitmentsDbContext : DbContext, IProviderCommitmentsDbCon
         public virtual DbSet<OverlappingEmail> OverlappingEmails { get; set; }
         public virtual DbSet<Learner> Learners { get; set; }
         public virtual DbSet<FlexibleEmployment> FlexibleEmployment { get; set; }
+        public virtual DbSet<ApprenticeshipPriorLearning> PriorLearning { get; set; }
```

Comment:
I think this may not be necessary - EF works out the tables based on the relationships between classes.  My understanding is that we only need to add a `DbSet` if we plan on querying the table directly, rather than simply including it in a join.